src/utils/excelService.ts

```
@@ -266,18 +266,9 @@ export class ExcelService {
-          if (typeof value === 'number') return value;
-          if (typeof value === 'string') {
-            const parsed = parseFloat(value.trim());
-            return isNaN(parsed) ? null : parsed;
-          }
-          return null;
-        };
-        
-        const area = parseNumberValue(row.getCell(areaIndex).value);
-        const length = parseNumberValue(row.getCell(lengthIndex).value);
-        const volume = parseNumberValue(row.getCell(volumeIndex).value);
+        const area = this.parseNumberValue(row.getCell(areaIndex).value);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)